notebooks/lensed_quasar_population_with_variability.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>can you comment a bit what's happening here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>